src/requests/user.js

```
@@ -250,6 +256,29 @@ const listOrganization = function (organizationIds = [], userToken = '') {
+			const internalToken = true
+
+			const tenantDetails = await requests.get(
+				tenantReadUrl,
+				'', // X-auth-token not required for internal call
+				internalToken
+			)
+			return resolve(tenantDetails)
+		} catch (error) {
+			return reject(error)
+		}
+	})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)